### ISSUE TITLE
package com.sothawo.mapjfx.event is not visible

### ISSUE THREAD

**Issue by vewert** [sothawo/mapjfx#73]
I am attempting to convert my application to use modules, but when I compile, I get the following error:
```
Error:(12, 26) java: package com.sothawo.mapjfx.event is not visible
  (package com.sothawo.mapjfx.event is declared in module com.sothawo.mapjfx, which does not export it)
```
In module-info.java I have: `requires com.sothawo.mapjfx;`

I'll admit, I am quite new (and not very knowledgeable about) java's module system, so I'm guessing I'm probably just missing something, but any help would be appreciated.

**Comment by sothawo** [comment]
I'll need to have a look at this; the demo app has a controller that uses the classes from events with no problems.

**Comment by sothawo** [comment]
will fix this this weekend

**Comment by vewert** [comment]
Thanks for looking into this.

Sorry, but I found that I am also seeing this with package: `com.sothawo.mapjfx.offline`

```
Error:(13, 26) java: package com.sothawo.mapjfx.offline is not visible
  (package com.sothawo.mapjfx.offline is declared in module com.sothawo.mapjfx, which does not export it)
```

I get it when importing: `com.sothawo.mapjfx.offline.OfflineCache;`

I'm guessing that this is a similar problem as the `event` package

**Comment by sothawo** [comment]
fixed in 2.12.0
